src/commands/plugins/source.ts

```diff
@@ -29,9 +29,9 @@ export default class PluginsSource extends Command {
   plugins = new Plugins(this.config);
 
   async run() {
-    const remoteCommands: string[] = FeatureFlag.getSingleton()
-      .getFeatureFlagInfo<string[]>("REMOTE_COMMANDS")
-      .sort();
+    const remoteCommandsDescriptions: Record<string, string> = FeatureFlag.getSingleton()
```

Comment:
What do you think about having something like this as the remote config? It seems more meaningful to me than a map of command and description.
```json
    "REMOTE_COMMANDS": [
       {
          "name": "add", 
          "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
        },
    ]
```

Reply:
I think it is a good idea. Thank you!